components-compose/src/main/res/drawable/components_select_row_circle.xml

```diff
@@ -0,0 +1,4 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Do we need to add this?

Reply:
No, actually these are not needed currently. I have copied them from the component module maybe we need these later.

Reply:
In general we should try and avoid adding stuff before we need it, as if for whatever reason we don't end up needing it then we've got unused files we need to go back and tidy up at some point. I think we can just add them when we need them and remove them for now. :)